docs/overview.md

```diff
@@ -4,6 +4,8 @@ Monokle is laid out like many other desktop applications:
 
 ![Monokle UI](img/monokle-ui-1.4.0.png)
 
+(On start-up, Monokle automatically loads the most recently selected folder or cluster.)
```

Comment:
Monokle will load the last opened folder only (not cluster) - and only if the corresponding setting is selected under Settings

Reply:
I just took this out here since it is explained in the "Settings" section.

Reply:
i meant removing the "or cluster." bit..